satpy/etc/readers/clavrx.yaml

```diff
@@ -1,10 +1,6 @@
 reader:
-  name: clavrx
-  short_name: CLAVR-X
-  long_name: The Clouds from AVHRR Extended (CLAVR-x)
   description: CLAVR-X Reader
-  status: Nominal
-  supports_fsspec: false
+  name: clavrx
```

Comment:
Is the removal of the metadata here intentional?

Reply:
Should I pull in upstream/main before the next commit?

Reply:
I'm not sure what pulling in upstream matters here. I guess maybe your base `main` branch was old? You should maybe merge upstream pytroll `main` into your PR. That should also fix the experimental CI environment that is failing.

Reply:
No.  That was not intentional.